Jenkinsfile

```diff
@@ -394,8 +386,9 @@ def describeBuild(buildContext) {
   }
 }
 
-def scheduleTriggerJobName() {
-  "drivers/python/oss/master/disabled"
+// branch pattern for cron
+def branchPatternCron() {
+  ~"(master)"
```

Comment:
you could change this to `jenkinsfile_cleanup` to test things before merging this PR if you want to